pkgs/data/fonts/orbitron/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "orbitron-${version}";
+  version = "20110526";
+
+  src = fetchFromGitHub {
+    owner  = "theleagueof";
+    repo   = "orbitron";
+    rev    = "13e6a52";
+    sha256 = "1c6jb7ayr07j1pbnzf3jxng9x9bbqp3zydf8mqdw9ifln1b4ycyf";
+  };
+
+  phases = [ "unpackPhase" "installPhase" ];
+
+  installPhase = ''
+    otfdir=$out/share/fonts/opentype/orbitron
+    ttfdir=$out/share/fonts/ttf/orbitron
+    mkdir -p $otfdir $ttfdir
+    cp -v Orbitron*.otf $otfdir
+    cp -v Orbitron*.ttf $ttfdir
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://www.theleagueofmoveabletype.com/orbitron";
+    downloadPage = "https://www.theleagueofmoveabletype.com/orbitron/download";
+    description = ''
+     Orbitron is a geometric sans-serif typeface intended for display
```

Comment:
This description is still way too long. What about simply "A geometric sans-serif font" or something to that effect but not very much longer (note that descriptions shall not contain the name of the package or trailing punctuation).